### PR TITLE
Dashboard restyling

### PR DESCRIPTION
**Pull request by eliorivero**
This PR introduces the following changes:

- updates the design of VaultPress UI so it's aligned with the current Jetpack branding.
- stylize all UI messages consistently
- introduce the Jetpack logo package and use it under the footer
- make notice style consistent throughout the WP Admin
- add core class to the notice so it's properly moved by WP core below the heading

Captura de Pantalla 2019-06-05 a la(s) 10 52 32

### CODE REVIEW

**Comment by enejb** [comment]
Thank you for working on this! It is starting to look great! 

I know this might be a bit more work but I think it would be pretty great to use some sort of admin page package. This package could be reused in a few different places and make it much easier to create such pages in current and future plugins. 

Jetpack Beta Plugin would be a great candidate for this. As well as Akismet plugin.

There is already some code in Jetpack that can be used to create more static type admin pages. See https://github.com/Automattic/jetpack/blob/master/_inc/lib/admin-pages/class.jetpack-admin-page.php#L244

**Comment by eliorivero** [comment]
We're now using the Jetpack logo package

Captura de Pantalla 2019-05-31 a la(s) 17 55 16

**Comment by eliorivero** [comment]
Notices throughout the WP Admin now share a common style

Captura de Pantalla 2019-06-05 a la(s) 10 40 26

Captura de Pantalla 2019-06-05 a la(s) 10 40 01

File: vaultpress.php [hunk 1]
```diff
@@ -306,7 +306,7 @@ function load_menu() {
 		if ( class_exists( 'Jetpack' ) ) {
 			$hook = add_submenu_page( 'jetpack', 'VaultPress', 'VaultPress', 'manage_options', 'vaultpress', array( $this, 'ui' ) );
 		} else {
-			$hook = add_menu_page( 'VaultPress', 'VaultPress', 'manage_options', 'vaultpress', array( $this, 'ui' ), 'div' );
+			$hook = add_menu_page( 'VaultPress1', 'VaultPress', 'manage_options', 'vaultpress', array( $this, 'ui' ), 'div' );
```
Comment (dereksmart) [inline vaultpress.php]: Is this `VaultPress1` change intentional?

File: vaultpress.php [hunk 2]
```diff
@@ -416,56 +416,89 @@ function error_notice() {
...
+	 *     An array of options to render the dashboard.
+	 *
+	 *     @type string $ui             Dashboard markup.
+	 *     @type string $dashboard_link Whether to show the link to the VaultPress dashboard.
+	 * }
+	 */
+	function ui_render() {
+		ob_start();
+
+		if ( !$this->is_localhost() ) {
```
Comment (dereksmart) [inline vaultpress.php]: This logic is backwards.

File: vaultpress.php [hunk 3]
```diff
@@ -563,79 +596,142 @@ function ui_load() {
...
-			$response = base64_decode( $this->contact_service( 'plugin_ui' ) );
-			echo $response;
+	/**
+	 * Renders the footer.
+	 */
+	function ui_footer() {
 		?>
+		<div class="jp-footer">
+			<div class="jp-footer__a8c-attr-container">
+				<a href="http://localhost/wp-admin/admin.php?page=jetpack_about">
```
Comment (dereksmart) [inline vaultpress.php]: `localhost` probably not intentional ;)

File: vaultpress.php [hunk 4]
```diff
@@ -563,79 +596,142 @@ function ui_load() {
...
+					</a>
+				</li>
+				<li class="jp-footer__link-item">
+					<a href="https://wordpress.com/tos/" class="jp-footer__link" title="<?php esc_attr_e( 'Terms of service', 'vaultpress' ) ?>" target="_blank" rel="noopener noreferrer">
+						<?php esc_html_e( 'Terms', 'vaultpress' ); ?>
+					</a>
+				</li>
+			</ul>
+			<div class="jp-power">
+				<span class="jp-power__text"><?php esc_html_e( 'Powered by', 'vaultpress') ?></span> <?php echo $this->ui_logo(); ?>
```
Comment (dereksmart) [inline vaultpress.php]: Let's link this to something - https://jetpack.com?

Bonus points for making it smart and linking to Jetpack dashboard if the plugin is active.

File: vaultpress.php (hunk 2 above)
Comment (eliorivero) [reply vaultpress.php]: It's fixed now. Thanks.

File: vaultpress.php (hunk 1 above)
Comment (eliorivero) [reply vaultpress.php]: Nope. I must've typed it somehow.

File: vaultpress.php (hunk 3 above)
Comment (eliorivero) [reply vaultpress.php]: Too much copy pasta 😅
Fixing it now.

File: vaultpress.php (hunk 4 above)
Comment (eliorivero) [reply vaultpress.php]: Done!

**Comment by eliorivero** [comment]
All updated!

**Comment by dereksmart** [comment]
@eliorivero I have rebased this against master to get the new Logo version

**Comment by eliorivero** [comment]
Updated with fix to hide the VP heading served remotely with certain UI messages.

The styles for these messages will remain as they are for now, since they have to be updated in the VP server. They integrate well and don't break the layout.

**Comment by oskosk** [comment]
Looking awesome.

I saw some details on mobile:

* The hamburger menu overlaps the page layout
* The text "The vaultpress plugin requires a subscription" wraps over the image.

image

**Comment by eliorivero** [comment]
Fixed layout of top notice

### Before

Captura de Pantalla 2019-06-11 a la(s) 19 11 50


### After

Captura de Pantalla 2019-06-11 a la(s) 19 11 29

**Pull request merged**